qemu_mode/build_qemu_support.sh

```diff
@@ -76,6 +76,18 @@ for i in libtool wget python automake autoconf sha384sum bison flex iconv patch
 
 done
 
+PYTHON_TEMP_SYMLINK=0
+if ! command -v python 1>/dev/null; then
+  if command -v python3 1>/dev/null; then
+    echo "[*] 'python' not found. 'python3' was found. Creating temporary symlink."
+    update-alternatives --install /usr/bin/python python "$(command -v python3)" 0 1>/dev/null
```

Comment:
this does not work. the user might not be root or /usr might be mounted read-only.
does alias work? e.g. `alias python=python3` ?
if that does not work create a symlink in the qemu directory and set the search path to look there first. on errors and exit that symlink would need to be removed though